article_store/articles/tests/conftest.py

```diff
@@ -0,0 +1,134 @@
+    return parse_fixture_file('0065_en_front.xml')
+
+
+@pytest.fixture(scope='session')
+def article_0065_en_front_payload_xml() -> str:
+    return parse_fixture_file('0065_en_front_payload.xml')
+
+
+@pytest.fixture(scope='session')
+def article_0065_es_front_xml() -> str:
+    return parse_fixture_file('0065_es_front.xml')
+
+
+@pytest.fixture(scope='session')
+def article_0065_es_front_payload_xml() -> str:
+    return parse_fixture_file('0065_es_front_payload.xml')
+
+
+@pytest.fixture(scope='session')
+def article_0065_pt_front_xml() -> str:
+    return parse_fixture_file('0065_pt_front.xml')
+
+
+@pytest.fixture(scope='session')
+def article_0065_pt_front_payload_xml() -> str:
+    return parse_fixture_file('0065_pt_front_payload.xml')
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def article_version_1(article: Article) -> ArticleVersion:
+    return ArticleVersion.objects.create(article=article, version=1)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def article_version_2(article: Article) -> ArticleVersion:
+    return ArticleVersion.objects.create(article=article, version=2)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def article_version_3(article: Article) -> ArticleVersion:
+    return ArticleVersion.objects.create(article=article, version=3)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def content_en_front(article_version_1: ArticleVersion,
+                     article_0065_en_front_payload_xml: str) -> Content:
+    return Content.objects.create(article_version=article_version_1,
+                                  language='en',
+                                  name='front',
+                                  model='http://localhost:8081/schemas/scielo/article/front.rng',
+                                  content=article_0065_en_front_payload_xml)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def content_es_front(article_version_1: ArticleVersion,
+                     article_0065_es_front_payload_xml: str) -> Content:
+    return Content.objects.create(article_version=article_version_1,
+                                  language='es',
+                                  name='front',
+                                  model='http://localhost:8081/schemas/scielo/article/front.rng',
+                                  content=article_0065_es_front_payload_xml)
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def content_pt_front(article_version_1: ArticleVersion,
+                     article_0065_pt_front_payload_xml: str) -> Content:
+    return Content.objects.create(article_version=article_version_1,
+                                  language='pt',
+                                  name='front',
+                                  model='http://localhost:8081/schemas/scielo/article/front.rng',
+                                  content=article_0065_pt_front_payload_xml)
+
+
+@pytest.fixture
+def content_xml_data() -> str:
+    return '''
+    <root>
+        <content-list>
+            <list-item>
```

Comment:
These three levels should be replaced with something like `<article><parts>`, but we'll sort it out later.